lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx

```diff
@@ -896,6 +901,13 @@ protected:
     {
       // Nothing to do
     }
+    PDFWrapper(const DistributionImplementation & distribution)
+      : EvaluationImplementation()
+      , p_distribution_(NULL)
+      , p_shared_distribution_(distribution.clone())
+    {
+      p_distribution_ = p_shared_distribution_.get();
```

Comment:
there should not be 2 kinds of pointer attribute; just keep one (also in other wrappers)

Reply:
also if we want to expose these via the new accessors the serialization should be implemented
and string operators could be implemented, currently it should be the default

Reply:
I removed the simple pointer and kept the shared pointer. 

Now __str__() yields things like 

```
'PDFWrapper(Normal(mu = 4, sigma = 3))'
```